README.md

```diff
@@ -123,6 +123,8 @@ can be sure that we have at least one version of the IO (and its COs) saved.
         * Call `createObjects` on the OCFL repository in order to:
             * overwrite the current file stored at the `destinationPath` provided
             * add a new version to the OCFL repository
+* Once these files are added to the OCFL repository, they can be deleted from the tmp directory in order to reduce space
```

Comment:
Can you say that they're deleted from the `work` directory. `tmp` is a specific thing and we're not storing them there any more.